src/thenewboston/serializers/network_block.py

```diff
@@ -40,6 +40,12 @@ def validate(self, data):
         if account_number in recipient_set:
             raise serializers.ValidationError('Block account_number not allowed as Tx recipient')
 
+        fee_list = [tx.get('fee', None) for tx in txs]
+        fee_set = set(fee_list)
+
+        if len(fee_list) != len(fee_set):
+            raise serializers.ValidationError('Tx fees must be unique')
+
```

Comment:
This logic won't quite work since there could be multiple non-fee Txs. In this case the `fee_list` would have multiple `None` items but would still be valid.